app/code/Magento/Customer/Block/Form/Register.php

```diff
@@ -220,4 +244,35 @@ public function getRequiredCharacterClassesNumber()
     {
         return $this->_scopeConfig->getValue(AccountManagement::XML_PATH_REQUIRED_CHARACTER_CLASSES_NUMBER);
     }
+
+    /**
+     * Determine if Address Fields have to be shown
+     *
+     * @return bool
+     */
+    public function getShowAddressFields()
```

Comment:
where is this method used?

Reply:
@vrann this method is used in the template it was already there but not in use